cypress/e2e/filter/filterCases.cy.ts

```diff
@@ -136,6 +136,99 @@ describe("Case list", () => {
       cy.get("tr").not(":first").get("td:nth-child(1)").contains("Alfred Pennyworth")
     })
 
+    it("Should display cases filtered by court name", () => {
+      cy.task("insertCourtCasesWithFieldOverrides", {
+        keywords: { courtNames: ["Manchester Court", "London Court", "Bristol Court"] },
+        force: "011111"
+      })
+
+      cy.visit("/bichard")
+
+      cy.get("button[id=filter-button]").click()
+      cy.get("input[id=court-name]").type("Manchester Court")
+
+      cy.get("button[id=search]").click()
+      cy.get("tr").not(":first").get("td:nth-child(3)").contains("Manchester Court")
```

Comment:
I'm wondering if it would be better to have `contains` on its own. Are the rows so important here? It would be easier to read and also if we add a new line in the future, it wouldn't break all the tests.

Reply:
I agree. I think we can do a follow-up PR in which we can refactor all the filter tests! 🚀